### ISSUE TITLE
ERROR TypeError: Cannot read property 'nativeElement' of undefined

### ISSUE THREAD

**Issue by kaname-png** [Mawi137/ngx-image-cropper#455]
ngrx-image-cropper version: 4.0.0
Angular: v12.1.1
Browser: Edge 91.0.864.64
When it happens: All the time
What to do to produce the error: Nothing, the error simply appears when the component is rendered.

> Code
```ts
import { Component } from '@angular/core';
import { ImageCroppedEvent, LoadedImage } from 'ngx-image-cropper';

@Component({
  selector: 'app-change-avatar',
  templateUrl: './change-avatar.component.html',
  styleUrls: ['./change-avatar.component.less'],
})
export class Component {
  constructor() {}

  imageChangedEvent: any = '';
  croppedImage: any = '';

  fileChangeEvent(event: any): void {
    this.imageChangedEvent = event;
  }
  imageCropped(event: ImageCroppedEvent) {
    this.croppedImage = event.base64;
  }
  imageLoaded(image: LoadedImage) {
    // show cropper
  }
  cropperReady() {
    // cropper ready
  }
  loadImageFailed() {
    // show message
  }

} 
```

>  Template
```html
<input type="file" (change)="fileChangeEvent($event)" />

<image-cropper
  [imageChangedEvent]="imageChangedEvent"
  [maintainAspectRatio]="true"
  [aspectRatio]="4 / 3"
  format="png"
  (imageCropped)="imageCropped($event)"
  (imageLoaded)="imageLoaded($event)"
  (cropperReady)="cropperReady()"
  (loadImageFailed)="loadImageFailed()"
></image-cropper>

<img [src]="croppedImage" />
```

> Error Stack
```ts
core.js:6456 ERROR TypeError: Cannot read property 'nativeElement' of undefined
    at CropperPositionService.resetCropperPosition (ngx-image-cropper.js:994)
    at ImageCropperComponent.resetCropperPosition (ngx-image-cropper.js:1659)
    at ImageCropperComponent.ngOnChanges (ngx-image-cropper.js:1382)
    at ImageCropperComponent.rememberChangeHistoryAndInvokeOnChangesHook (core.js:1498)
    at callHook (core.js:2536)
    at callHooks (core.js:2495)
    at executeInitAndCheckHooks (core.js:2446)
    at selectIndexInternal (core.js:8428)
    at Module.ɵɵadvance (core.js:8411)
    at ChangeAvatarComponent_Template (change-avatar.component.html:14)
```

**Comment by Bond1768** [comment]
I have the same problem

**Comment by Bond1768** [comment]
Version 3.0.0 does not have this problem.

**Comment by Enngage** [comment]
Yeah, same problem here after updating to `4.0.0`

**Comment by MachavarapuRamya** [comment]
I'm also facing the same issue.

**Comment by danielfsc** [comment]
This error does not break the lib. But I fix this by changing the method resetCropperPosition in the cropper-position.service.ts file. I just added:
`if (!sourceImage?.nativeElement) {
      return;
    }`

**Comment by kristher1619** [comment]
Also facing this problem

**Comment by Sergiobop** [comment]
I downgraded to 3.3.5 meanwhile

**Comment by stefanhk31** [comment]
We are also encountering this issue.

**Comment by casper5822** [comment]
Same for me, version 4 with angular 11

**Comment by Sergiobop** [comment]
pinging @Mawi137 in case he didn't notice this issue

**Comment by sobelfall** [comment]
Idem for me. So note that is not blocked

**Comment by Mawi137** [comment]
Hi all, this is fixed in version 4.0.1 as suggested by @danielfsc. Sorry for the inconvenience!
